setup.py

```diff
@@ -45,6 +57,9 @@ def find_tests_require():
         'Programming Language :: Python',
         'Programming Language :: Python :: 2.6',
         'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3.3',
+        'Programming Language :: Python :: 3.4',
+        'Programming Language :: Python :: 3.5',
```

Comment:
This should include:

```
Programming Language :: Python :: 2
```

and:

```
Programming Language :: Python :: 3
```

This is mostly to be nice to some of the tools out there tracking python 2 vs. python 3 support. Otherwise I don't think it affects anything.